v2/reader.go

```diff
@@ -2,6 +2,7 @@ package car
 
 import (
 	"fmt"
+	blocks "github.com/ipfs/go-block-format"
```

Comment:
nit: don't put third party imports in the std group. I think gofumpt fixes this.